File: apps/problems/arraySum/boilerplate-full/function.js

##USER_CODE_HERE##

const input = require('fs').readFileSync(0, 'utf-8').trim().split(/\s+/);
const size_arr = parseInt(input.shift());
const arr = input.splice(0, size_arr).map(Number);
const result = Sum(arr);
console.log(result);
    

File: apps/problems/arraySum/boilerplate-full/function.rs

use std::io::{self, Read};

##USER_CODE_HERE##

fn main() {
  let mut input_str = String::new();
  io::stdin().read_to_string(&mut input_str).unwrap();
  let mut input = input_str.split_whitespace();
  let size_arr: usize = input.next().unwrap().parse().unwrap();
  let arr: Vec<i32> = (0..size_arr).map(|_| input.next().unwrap().parse().unwrap()).collect();
  let result = Sum(arr);
  println!("{}", result);
}
    